server/src/routes/index.ts

```diff
@@ -39,7 +39,24 @@ router.get('/api-docs', swaggerUI.setup(openapiSpecification));
 
 /**
  * @swagger
+ * components:
+ *   securitySchemes:
+ *     jwtCookie:
+ *       type: apiKey
+ *       in: cookie
+ *       name: jwt
  * definitions:
+ *   Failure:
+ *     type: object
+ *     description: A reason for a failure, with a chiefly developer-facing error message
+ *     properties:
+ *       success:
+ *         type: boolean
+ *         description: '`false` to indicate failure'
```

Comment:
Quotation marks.

Reply:
These quotation marks turned out to be necessary, as otherwise YAML doesn't allow string literals that start with a backtick.